static/app/views/starfish/views/appStartup/screenSummary/index.tsx

```diff
@@ -31,6 +31,7 @@ import {ScreenLoadSpanSamples} from 'sentry/views/starfish/views/screens/screenL
 import AppStartWidgets from './widgets';
 
 type Query = {
+  ['device.class']: string;
```

Comment:
This would also work:

```suggestion
  'device.class': string;
```